src/js/gi/components/Checkbox.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import _ from 'lodash';
+
+import ToolTip from '../../common/components/form-elements/ToolTip';
+
+
+/**
+ * A form checkbox with a label that can display error messages.
```

Comment:
* the concepts of required fields and error messages do not apply to GIBCT
* click handlers on the common components also seemed like overkill and I frankly understand why they were implemented the way they are, i.e. `{value: x, dirty: true}`. I'm sure there's a good reason but it felt like the reason was maybe only well-suited to more complicated forms and schema forms elsewhere on the site.